Styles/mobilemain.css

```diff
@@ -48,29 +52,19 @@
         content: "";
         display: block;
         width: 40px;
-        line-height: 5px;
-        margin-right: auto;
-        margin-left: auto;
         border-top: 5px solid #3A606E;
-        margin-top: 1em;
-        cursor: pointer;
     }
     .burger::after {
         position: absolute;
         content: "";
         display: block;
-        line-height: 5px;
         width: 40px;
-        margin-right: auto;
-        margin-left: auto;
         border-top: 5px solid #3A606E;
-        margin-top: 1em;
-        cursor: pointer;

```

Comment:
You don't need these declarations to have the same visual appearance of the burger icon